// @fag/felix/felix.esm.js
var colors = {
  gray: {
    '900': '#0a0a09',
    '800': '#141411',
    '700': '#2a2b26',
    '600': '#53544c',
    '500': '#7f8076',
    '400': '#aaaba0',
    '300': '#d3d4c9',
    '200': '#ebebe2',
    '100': '#f5f5ef'
  },
  red: {
    '900': '#290607',
    '800': '#470a0d',
    '700': '#710e15',
    '600': '#a11420',
    '500': '#cb1d37',
    '400': '#ed3467',
    '300': '#fc64a1',
    '200': '#ffa3d6',
    '100': '#ffecfa'
  },
  orange: {
    '900': '#2f150d',
    '800': '#50240f',
    '700': '#75330f',
    '600': '#9e490c',
    '500': '#cb6709',
    '400': '#f1901d',
    '300': '#fcb760',
    '200': '#fed7a8',
    '100': '#fff6e9'
  },
  yellow: {
    '900': '#371f05',
    '800': '#5f3906',
    '700': '#8f5c07',
    '600': '#c6930b',
    '500': '#f8cb1b',
    '400': '#f7dd4b',
    '300': '#f5ea7e',
    '200': '#f3f1b0',
    '100': '#f2f2de'
  },
  green: {
    '900': '#052907',
    '800': '#0c4a07',
    '700': '#1e6e08',
    '600': '#3f9608',
    '500': '#6fbf10',
    '400': '#a5e22c',
    '300': '#cfed6d',
    '200': '#e4efab',
    '100': '#e8ead8'
  },
  blue: {
    '900': '#001336',
    '800': '#002662',
    '700': '#013f8c',
    '600': '#0362b2',
    '500': '#0f8fd0',
    '400': '#38bce6',
    '300': '#70dbef',
    '200': '#a9eaf4',
    '100': '#e5f6f8'
  },
  purple: {
    '900': '#120639',
    '800': '#271163',
    '700': '#442294',
    '600': '#643bc0',
    '500': '#8659e1',
    '400': '#9f7af2',
    '300': '#b29af8',
    '200': '#ccbdf9',
    '100': '#f0e6f9'
  }
};

// import './colors.css';
var sum = function sum(a, b) {
  if ('development' === process.env.NODE_ENV) {
    console.log('boop');
  }

  return a + b;
};

export { sum, colors as tailwindConfig };
